src/main/kotlin/lotto/domain/LottoNumber.kt

```diff
@@ -0,0 +1,12 @@
+package lotto.domain
+
+data class LottoNumber(val number: Int) {
+    init {
+        require(number in MIN_VALUE..MAX_VALUE) { "Lotto number must be between 1 and 45." }
```

Comment:
```suggestion
        require(number in MIN_VALUE..MAX_VALUE) { "Lotto number must be between $MIN_VALUE and $MAX_VALUE." }
```

I think the method above would be more flexible to changes.